### DISCUSSION TITLE
Is there some option similar to removed APPLY_AUTO_IMPORT_NAMES_ON_CHANGED_FILES_ONLY?

### DISCUSSION THREAD

**Discussion by markoph** [rectorphp/rector#8229] [Q&A] [answered]
Hi,

we wrote some new rules for our app and had to update rector. We found out that `APPLY_AUTO_IMPORT_NAMES_ON_CHANGED_FILES_ONLY` was removed (https://github.com/rectorphp/rector-src/commit/a4e7a4371a1a85f2dd3fcb9ec7d9de2a8a04d3b9).

Is there some other way how to not change imports in files other rules didn't touch?

Thank you.

**Comment by TomasVotruba** [comment, marked as answer]
Hi, we removed this option because it changed files in rather unexpected way. There is no replacement.
Instead it's better to apply coding standard first - import all FQN names, and only then apply next rules.

**Reply by markoph** [reply]
OK. Thank you for answer :+1:
